lobsterpy/cli.py

```diff
@@ -36,108 +37,145 @@ def get_parser() -> argparse.ArgumentParser:
-    input_parent = argparse.ArgumentParser(add_help=False)
-    input_file_group = input_parent.add_argument_group("Input files")
-    input_file_group.add_argument(
-        "--poscar",
-        "--POSCAR",
-        dest="poscar",
-        default="POSCAR",
+    # Arguments that are needed by different actions, but not always
+
+    incar_file = argparse.ArgumentParser(add_help=False)
+    incar_file.add_argument(
+        "--file-incar",
+        default="INCAR",
+        dest="incar",
         type=Path,
-        help='path to POSCAR. Default is "POSCAR"',
+        help='path to INCAR. Default is "INCAR".',
     )
-    input_file_group.add_argument(
-        "--charge",
+
+    charge_file = argparse.ArgumentParser(add_help=False)
+    charge_file.add_argument(
+        "--file-charge",
         default="CHARGE.lobster",
+        dest="charge",
         type=Path,
         help='path to CHARGE.lobster. Default is "CHARGE.lobster"',
     )
-    input_file_group.add_argument(
-        "--icohplist",
-        default="ICOHPLIST.lobster",
-        type=Path,
-        help='path to ICOHPLIST.lobster. Default is "ICOHPLIST.lobster"',
-    )
-    input_file_group.add_argument(
-        "--cohpcar",
-        default="COHPCAR.lobster",
-        type=Path,
-        help='path to COHPCAR.lobster. Default is "COHPCAR.lobster". This argument'
-        "will also be read when COBICARs or COOPCARs are plotted.",
-    )
-    input_file_group.add_argument(
-        "--incar",
-        default="INCAR",
+
+    structure_file = argparse.ArgumentParser(add_help=False)
+    structure_file.add_argument(
+        "--file-structure",
+        default="POSCAR",
+        dest="structure",
         type=Path,
-        help='path to INCAR. Default is "INCAR".',
+        help='path to structure file. Default is "POSCAR"',
     )
-    input_file_group.add_argument(
-        "--potcar",
+
+    potcar_file = argparse.ArgumentParser(add_help=False)
+    potcar_file.add_argument(
+        "--file-potcar",
         default="POTCAR",
+        dest="potcar",
         type=Path,
         help='path to POTCAR. Default is "POTCAR".',
     )
-    input_file_group.add_argument(
-        "--potcar-symbols",
-        dest="potcarsymbols",
-        type=_potcar_symbols,
-        # nargs="+",
-        help="List of potcar symbols",
-    )
 
-    input_file_group.add_argument(
-        "--lobsterin",
-        default="lobsterin",
+    icohplist_file = argparse.ArgumentParser(add_help=False)
+    icohplist_file.add_argument(
+        "--file-icohplist",
+        default="ICOHPLIST.lobster",
+        dest="icohplist",
         type=Path,
-        help='path to lobsterin. Default is "lobsterin".',
+        help='path to ICOHPLIST.lobster. Default is "ICOHPLIST.lobster"',
     )
 
-    input_file_group.add_argument(
-        "--lobsterout",
-        default="lobsterout",
+    coxxcar_file = argparse.ArgumentParser(add_help=False)
+    coxxcar_file.add_argument(
+        "--file-cohpcar",
```

Comment:
While this is cleaner, we should leave cohpcar as an alias in any case.

Reply:
Should we have such an option for all others as well? 

I was thinking maybe I add a shorthand alias like `-fcohp` for all the file-related args. Could be much more intuitive?

Reply:
-fcohp sounds good